docs/source/api/apollo-server.mdx

```diff
@@ -668,8 +515,7 @@ const result = await server.executeOperation({
 The `executeOperation` method takes two arguments:
 * The first argument is an object describing the GraphQL operation to be executed.
   * Supported fields are listed in the table below.
-* The second optional argument is passed to `ApolloServer`'s `context` function.
-  * This is helpful when [testing](../testing/testing/) that your server's `context` is correctly gathering the values needed from a request.
+* The second optional argument is used as the operation's [context value](../data/resolvers/#the-context-argument).
 
 Below are the available fields for the first argument of `executeOperation`:
```

Comment:
You can also include `extensions` and `http`. `http` has the same structure as the main option to `executeHTTPGraphQLRequest`. These are generally only interesting if you have a plugin that looks at those fields on `requestContext.request`.

Reply:
Do you happen to know what the `extensions` field is used for? I couldn't find a clear answer in the `GraphQLRequest` interface, and I'm not sure where else to look for more info on that field.

Reply:
`extensions` is an object of arbitrary data that can be used to pass information along from one touch point to another. It's effectively a context object but the scope is usually meant to be much smaller (generally pertaining to the object the `extensions` object is attached to).

Reply:
@trevor-scheer How does this feel for an explanation of `extensions`:

> The `extensions` object is usually used to pass information within your server from one touch point to another (e.g., from a request to a plugin).